pkgs/by-name/in/instant-scala/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  scala-cli,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "instant-scala";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jatcwang";
+    repo = "instant-scala";
+    rev = "/refs/tags/v${finalAttrs.version}";
+    sha256 = "jqSvKTL8NzqjwqDj/+55YWecx2bnzuArP8RdfH5q/1U=";
+  };
+
+  buildInputs = [
+    scala-cli
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp instant-scala $out/bin/instant-scala
+    chmod +x $out/bin/instant-scala
```

Comment:
> Use `install` instead.

Sorry, my metallic friend.. You'd use `install` if there was an installer script or makefile, but the repo has a plain bash script. The install phase, as written, looks correct.

From the [manual](https://nixos.org/manual/nixpkgs/stable/#ssec-install-phase):
"The default installPhase creates the directory `$out` and calls `make install`"

Since there's no makefile, creating `$out` and copying appropriately is the right answer.

Reply:
Bot is right, use `install -Dm755 instant-scala -t $out/bin` instead.

Reply:
@Bot-wxt1221 my apologies. TIL.